data/quarg/quarg missions.txt

```diff
@@ -113,3 +113,134 @@ mission "First Contact: Kuwaru Efreti"
+				`	"I don't have the time, no."`
+					decline
+			`	"To help humans in the other end of this region, the South, be better prepared against incoming attacks, and help them keep track of ships that pass by their systems."`
+			choice
+				`	"I'd like to help with that then. Where am I deploying them?"`
+				`	"I actually am headed to another part of the galaxy now, sorry."`
+					decline
+			label where
+			`	"To <waypoints>. Once you are done, return here and you will receive <payment>."`
+			`	When you're back at your ship some more Quarg are ready with some robots, putting the four satellites inside your ship. They have some negative space, and are pyramidical, with each end having smaller pyramids on them. Each of the devices is roughly half your size, but probably weigh a lot more than you.`
+				accept
+	on enter "Kappa Centauri"
+		dialog "You open your cargo hold, and let a Quarg scanner drift out to the void."
+	on enter "Graffias"
+		dialog "Looking at the former slave-colony planet here as you deploy their scanner, you wonder if the Quarg heard about Poisonwood's history, and that's why decided to help with surveillance."
+	on enter "Antares"
+		dialog "You set a Quarg scanner free, and let it leave your ship."
+	on enter "Han"
+		dialog "Although those that live on the planet here don't seem to mind the occasional pirates demanding tribute, maybe they will come to appreciate the reduced pirate presence here, assuming the Quarg scanner really does help with that."
+	on complete
+		payment 263000
+		dialog `You land to find a Quarg waiting right at the docks. It hands you <payment>, and thanks you for your help.`
+
+
+
+mission "Lagrange Ring: Cargo"
+	minor
+	name "Humane Construction Materials"
+	description "Head to <stopover> to pick up <cargo>, then bring it all to <destination>, where the Quarg are preparing to open the first section of their ringworld for human visit."
+	cargo "Construction Materials" 254
+	source
+		near "Enif" 1
+		government "Quarg"
+		not planet "Humanika"
+	stopover "Chiron"
+	destination "Lagrange"
+	to offer
+		has "First Contact: Quarg: offered"
+		"pirate attraction" <= 3
+	on offer
+		conversation
+			`In the spaceport, a group of Quarg is talking to dozens of human captains. Some nod and head right back to their ships, while a few others remain and appear to discuss a bit more with the Quarg.`
+			choice
+				`	(Join them to see what's going on.)`
+				`	(Ignore them.)`
+					decline
+			`	You join the group, and one of the Quarg diverts its attention to you, while others continue speaking with the human merchants.`
+			`	"Would you also be someone willing to provide us a cargo service, Captain?" It asks, all the while checking a data pad, it seems to be related to the ships docked here. "Not many humans in this region have the cargo space required, much less proper defenses for cargo convoys. We are willing to richly pay for whoever meets the criteria and is willing to help us."`
```

Comment:
1 : changing 
>"not many humans" 

into "not many of _you_  humans would make it seem more like it's a quarg talking

2 : changing 
>"willing to richly pay"

into "willing to pay adequately" , it makes more sense and it sounds more quarg-ish